js/components/frame.js

```diff
@@ -70,12 +71,24 @@ class Frame extends ImmutableComponent {
   }
 
   componentDidMount () {
+    ipc.on(messages.CERT_ERROR, (e, details) => {
```

Comment:
We used to have some ipc listener registration in frame.js but we had to move it out because creating lots of frames was giving there might be a leak warning. Might be worth checking for similar warnings with lots of tabs.